pkg/daemon/update.go

```diff
@@ -48,6 +48,12 @@ const (
 	fipsFile              = "/proc/sys/crypto/fips_enabled"
 	extensionsRepo        = "/etc/yum.repos.d/coreos-extensions.repo"
 	osImageContentBaseDir = "/run/mco-machine-os-content/"
+
+	// These are the actions for a node to take after applying config changes.
+	// Defaults to reboot.
+	postConfigChangeActionNone       = "none"
```

Comment:
What change would result in no action?

Reply:
SSH keys + pull secret. I'll add some comments here.

"No action" is perhaps not the right way to frame it, we still cordon, drain + write files/units/etc. We just don't reload any unit and don't reboot.

Reply:
maybe postConfigChangeActionSkipReboot? tho i think the none also works

Reply:
I think e.g. reloading crio also "skips reboot", so I defaulted to "none" to mean "no extra action needed"

Reply:
good point, im fine with none :+1: